root/programs/C#/Frameworks/Infrastructure/Public/Util/Latebind.cs

```diff
@@ -84,8 +85,7 @@ public static object InvokeMethod(string assemblyName, string className, string
             {
                 // アセンブリ ロード
                 string ext = ".dll";
-                int extLen = ext.Length;
-                if (assemblyName.Substring(assemblyName.Length - extLen, extLen) == ext)
+                if (assemblyName.EndsWith(ext))
```

Comment:
@daisukenishino 

We have modified the nishino branch directly with [this](https://github.com/OpenTouryoProject/OpenTouryo/pull/140/files#r49158443) chanages.